rmi/master.sh

```diff
@@ -0,0 +1,18 @@
+printf "
```

Comment:
bash this file and you get some ssh commands for copying/pasting, it doesn't actually do anything though